From 51f952a6e018049d4b52bbd419653745cf1aaab3 Mon Sep 17 00:00:00 2001
From: Behdad Esfahbod <behdad@behdad.org>
Date: Sun, 6 Jan 2013 23:15:44 -0600
Subject: [PATCH] Bug 691263 - Build with automake-1.13 broken

Replace long obsolete AM_CONFIG_HEADER with proper AC_CONFIG_HEADERS.
automake-1.13 errors out upon seeing former.

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 8362280..dcd115e 100644
--- a/configure.in
+++ b/configure.in
@@ -54,7 +54,7 @@ dnl -Wno-portability tells automake to stop complaining about the
 dnl usage of GNU Make specific features.
 AM_INIT_AUTOMAKE(1.9 gnits dist-xz no-dist-gzip -Wno-portability)
 m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
-AM_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADERS([config.h])
 
 
 PANGO_VERSION_MAJOR=pango_version_major()
-- 
1.8.2.1

